app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -32,6 +32,10 @@ import org.mozilla.fenix.helpers.ext.waitNotNull
 import org.mozilla.fenix.ui.robots.mDevice
 
 object TestHelper {
+    // Variable used to store currently running package name.
+    // Use this to replace hardcoded package name in UIAutomator matchers
+    val packageName = InstrumentationRegistry.getInstrumentation().targetContext.packageName
```

Comment:
Is this used anywhere?

Reply:
oops, meant to, but it wasn't needed anymore. Thanks for catching this!